packages/vm/src/cache.rs

```diff
@@ -132,16 +130,11 @@ where
 
     pub fn store_instance(
         &mut self,
-        checksum: &Checksum,
+        _checksum: &Checksum,
         instance: Instance<S, A, Q>,
     ) -> Option<Extern<S, A, Q>> {
-        if let Some(cache) = &mut self.instances {
-            let (wasmer_instance, ext) = Instance::recycle(instance);
-            cache.put(*checksum, wasmer_instance);
-            ext
-        } else {
-            None
```

Comment:
Ah, Ethan, by the way: for the default configuration (cache size 0) this never returned the Extern instance, which I think is a bug. I saw that the result of `store_instance` is not used in go-cosmwasm. How is it even possible to reuse deps in a safe way there?